external-crates/move/crates/move-command-line-common/src/types.rs

```diff
@@ -24,9 +24,14 @@ pub enum TypeToken {
 }
 
 #[derive(Eq, PartialEq, Debug, Clone)]
-pub struct ParsedStructType {
+pub struct ParsedModuleId {
     pub address: ParsedAddress,
-    pub module: String,
+    pub name: String,
+}
+
+#[derive(Eq, PartialEq, Debug, Clone)]
+pub struct ParsedStructType {
```

Comment:
Ok I see, so say we're looking at 0x2::coin::Coin, the new structure would be

ParsedStructType {
module: ParsedModuleId,
name: Coin,
type_args: Vec<T> // something like that
}

ParsedModuleId {
address: ParsedAddress,
name: coin
}

ParsedAddress remains unchanged